model/model-verifier/src/test/java/io/spine/model/verify/given/UploadCommandAssignee.java

```diff
@@ -26,15 +26,15 @@
 
 package io.spine.model.verify.given;
 
-import io.spine.server.command.AbstractCommandHandler;
+import io.spine.server.command.AbstractCommandAssignee;
 import io.spine.server.command.Assign;
 import io.spine.test.model.verify.command.UploadPhoto;
 import io.spine.test.model.verify.event.PhotoUploaded;
 
-public class DuplicateCommandHandler extends AbstractCommandHandler {
+public class UploadCommandAssignee extends AbstractCommandAssignee {
```

Comment:
Not sure why this name does not reflect the duplication anymore. Let's discuss.

Reply:
`UploadCommandAssignee` and `DuplicateCommandAssignee` have almost the same content, so GitHub mislinked the modified files here.